templates/components/blog_search.html.twig

```diff
@@ -0,0 +1,20 @@
+<div {{ attributes }}>
```

Comment:
same as above, adding link to `{# Lear more on https://symfony.com/bundles/ux-live-component/current/index.html #}`